apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,27 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "mbed.h"
+#include "TutorialServo.h"
+
+// Blinking rate in milliseconds
+// #define BLINKING_RATE 500ms
+// AnalogIn potVoltageIn(PA_0);
+// PwmOut servoPwmOut(PA_1);
+
+CAN can(CAN1_RX, CAN1_TX);
+
+TutorialServo servo(PA_1);
+CANMessage msg;
+
+
+int main() {    
+    while(true) {
+        if(can.read(msg)){
+          servo.setPositionInDegrees(msg.data[0]);
```

Comment:
The CAN message gives a percentage, you need to convert it to degrees

Reply:
so a better way to set the position would be as follows:
- declare a variable of type float
- use the msg.getpayload(var)
rather than passing msg.data[0] as a variable, you should pass the variable used in the getpayload
(let me know if this explanation doesnt make sense)